main/station-config.js

```diff
@@ -7,9 +7,12 @@ const ConfigKeys = {
   OnboardingCompleted: 'station.OnboardingCompleted',
   TrayOperationExplained: 'station.TrayOperationExplained',
   StationID: 'station.StationID',
-  FilAddress: 'station.FilAddress'
+  FilAddress: 'station.FilAddress',
+  DestinationFilAddress: 'station.FilAddress' // todo - replace by 'station.DestinationFilAddress'
```

Comment:
same question here, why not update this directly in this PR?

Reply:
check [previous comment](https://github.com/filecoin-station/filecoin-station/pull/333/files/2a08e231d6970334986225ee7159f8b35c2ec017#r1050771859) pls